scripts/rebuild_for_camera.sh

```diff
@@ -0,0 +1,27 @@
+#This script can be used to rebuild the SDK depending on various platforms and cameras.
+#Its purpose is to be called from a shortcut placed on desktop.
+#It expects the parameters passed as the CMAKE command.
+
+#Possible command for the camera: -DUSE_FXTOF1=1 
```

Comment:
`command` is not a good word for this. Maybe `build flag` or `argument` would be more appropriate.